src/main.py

```diff
@@ -2,3 +2,6 @@ class Foo(object):
 	def __init__(self):
 		self.x=1
 		self.y=3 # by duyhsieh
+
+	def feature1(self):
```

Comment:
what is the feature about ? nothing is inside and no comment at all.

Reply:
this is a test function only. no harm.